packages/blocks/src/_common/consts.ts

```diff
@@ -8,3 +10,19 @@ export const EDGELESS_BLOCK_CHILD_BORDER_WIDTH = 2;
 // The height of the header, which is used to calculate the scroll offset
 // In AFFiNE, to avoid the option element to be covered by the header, we need to reserve the space for the header
 export const PAGE_HEADER_HEIGHT = 53;
+
+export const LINK_CARD_WIDTH: Record<LinkCardStyle, number> = {
```

Comment:
Maybe `EMBED_CARD_WIDTH` would be more suitable.

Reply:
@Saul-Mirone 
This will be used for bookmark and linked pages as well, so can we have a common name that represents all the card ?

Reply:
> @Saul-Mirone This will be used for bookmark and linked pages as well, so can we have a common name that represents all the card ?

According to the discussions with @doodlewind . We should use `embed` to describe the components that display card information for a special kind of url. Since we already have `bookmark` block, there's no need to rename it to introduce breaking changes. But **embed** is precisely the word to describe this kind of blocks.

Reply:
@Saul-Mirone @doodlewind 
done